castervoice/lib/settings.py

```diff
@@ -242,14 +242,25 @@ def _deep_merge_defaults(data, defaults):
 
 
 def _get_defaults():
-    terminal_path_default = "C:/Program Files/Git/git-bash.exe"
+
+    if sys.platform == "win32":
+        terminal_path_default = "C:/Program Files/Git/git-bash.exe"
+    elif sys.platform.startswith("linux"):
+        terminal_path_default = "/usr/bin/gnome-terminal"
+    elif sys.platform == "darwin":
+        terminal_path_default = "/Applications/Utilities/Terminal.app"
+
     if not os.path.isfile(terminal_path_default):
         terminal_path_default = ""
 
     ahk_path_default = "C:/Program Files/AutoHotkey/AutoHotkey.exe"
```

Comment:
ahk_path_default seems to be windows os specific handled in other OSs for loading AutoHotkey?

Reply:
I have not found AHK to be necessary yet on linux